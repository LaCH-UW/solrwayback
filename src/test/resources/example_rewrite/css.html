<?xml version="1.1" encoding="iso-8859-1"?>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="da">
<head>
    <title>kimse.rovfisk.dk/katte/</title>
    <style type="text/css" media="screen">@import "/css/forum.css";</style>
    <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
    <link rel="stylesheet" href="/style.css"  type="text/css" media="screen" />
    <link rel="alternate" type="application/rss+xml" title="RSS 2.0" href="/rss2.php" />
</head>
<body>
<style type="text/css" media="screen">@import "//www.dr.dk/drdkGlobal/spot/spotGlobal.css";</style>
<style type="text/css" media="screen">@import "/design/www/global/css/globalPrint.css";</style>
<style type="text/css" media="screen">@import url(http://en.statsbiblioteket.dk/portal_css/SB%20Theme/resourceplonetheme.sbtheme.stylesheetsmain-cachekey-7e976fa2b125f18f45a257c2d1882e00.css);</style>
<a class="toplink" href="test" class="button" style="background:url(img/homeico.png) no-repeat ; width:90px; margin:0px 0px 0px 16px;">kimse.rovfisk.dk  </a>
<a class="toplink" href="/katte/">katte / </a><br /><br />
<table cellspacing="8"><tr><td></td><td class="itemw"><a href="/katte/?browse=DSC00175.JPG"><img class="lo" src="/cache/katte/DSC00175.JPG" /></a></td>
    <td></td><td class="itemw"><a href="/katte/?browse=DSC00209.JPG"><img class="lo" src="/cache/katte/DSC00209.JPG" /></a></td>
    <img  src="http://belinda.statsbiblioteket.dk:9721/solrwayback/services/downloadRaw?source_file_path=/netarkiv/0122/filedir/285330-279-20180113095114447-00002-kb-prod-har-001.kb.dk.warc.gz&amp;offset=685207223" srcset="http://skraedderiet.dk/wp-content/uploads/2018/01/saks_watermark-001_transparent.jpg 477w, http://skraedderiet.dk/wp-content/uploads/2018/01/saks_watermark-001_transparent-150x150.jpg 150w, http://skraedderiet.dk/wp-content/uploads/2018/01/saks_watermark-001_transparent-298x300.jpg 298w" sizes="(max-width: 235px) 100vw, 235px">
</table><br />  </body>
</html>"