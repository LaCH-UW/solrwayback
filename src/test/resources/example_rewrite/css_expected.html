<!--?xml version="1.1" encoding="iso-8859-1"?-->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="da">
 <head>
  <title>kimse.rovfisk.dk/katte/</title>
  <style type="text/css" media="screen">@import url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=5591);</style>
  <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
  <link rel="stylesheet" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=8821" type="text/css" media="screen">
  <link rel="alternate" type="application/rss+xml" title="RSS 2.0" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=9807">
 </head>
 <body>
  <style type="text/css" media="screen">@import url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=8253);</style>
  <style type="text/css" media="screen">@import url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=913);</style>
  <style type="text/css" media="screen">@import url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=1130);</style>
  <a class="button" href="http://localhost:0000/solrwayback/services/web/2020043030700/http://example.com/somefolder/test" style="background:url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=5711) no-repeat ; width:90px; margin:0px 0px 0px 16px;">kimse.rovfisk.dk </a>
  <a class="toplink" href="http://localhost:0000/solrwayback/services/web/2020043030700/http://example.com/katte/">katte / </a>
  <br>
  <br>
  <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=8704" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=1414 477w, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=8029 150w, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=1673 298w" sizes="(max-width: 235px) 100vw, 235px">
  <table cellspacing="8">
   <tbody>
    <tr>
     <td></td>
     <td class="itemw"><a href="http://localhost:0000/solrwayback/services/web/2020043030700/http://example.com/katte/?browse=DSC00175.JPG"><img class="lo" src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=8174"></a></td>
     <td></td>
     <td class="itemw"><a href="http://localhost:0000/solrwayback/services/web/2020043030700/http://example.com/katte/?browse=DSC00209.JPG"><img class="lo" src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=9662"></a></td>
    </tr>
   </tbody>
  </table>
  <br>  "
 </body>
</html>